pyquil/quil.py

```diff
@@ -73,6 +75,24 @@ def __iter__(self):
         """
         return self.actions.__iter__()
 
+    def get_qubits(self):
```

Comment:
I think that is mostly potayyto potahhto in this case.  The qubits that are allocated to a pyQuil program are a property of that program